plugins/network-elements/tungsten/src/main/java/org/apache/cloudstack/network/tungsten/api/command/ConfigTungstenFabricServiceCmd.java

```diff
@@ -132,6 +135,21 @@ public void doInTransactionWithoutResult(final TransactionStatus status) {
 
     private void persistDefaultSystemNetwork() {
         Transaction.execute(new TransactionCallbackNoReturn() {
+
+            private void persistNetworkServiceMapAvoidingDuplicates(Network network,
+                                                                    Network.Service service,
+                                                                    Network.Provider provider,
+                                                                    NetworkServiceMapVO mapVO) {
```

Comment:
@nvazquez isn't mapVO here already providing the service and provider? Do we need service and provider as an argument?

Reply:
Thanks @shwstppr, fixed